package.json

```
@@ -34,6 +34,7 @@
         "ag-grid-react": "^33.1.0",
         "clsx": "^2.1.1",
         "core-js": "^3.40.0",
+        "mui-nested-menu": "^4.0.1",
```

```suggestion
        "mui-nested-menu": "^4.0.0",
```
Make sure the same version number as in study, to facilitate the migration